app/views/shared/api_entreprise/_containerized_body.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="fr-container fr-mb-5w fr-mt-5w">
+<div class="fr-container<%= '-fluid' if @large_container%> fr-mb-5w fr-mt-5w">
```

Comment:
La joie de l'erb :')
C'est difficile à lire, je ferai plus ça:

```suggestion
<% container_klass = @large_container.present? ? 'fr-container-fluid' : 'fr-container' %>
<div class="<%= container_klass %> fr-mb-5w fr-mt-5w">
```

Le mieux potentiellement serait d'utiliser `content_tag` pour bien traduire qu'on a de la logique.

Reply:
what do you mean with content_tag ?

Reply:
Ceci https://apidock.com/rails/ActionView/Helpers/TagHelper/content_tag

Reply:
aaaah oké !